pkg/cmd/serve.go

```diff
@@ -78,6 +78,9 @@ func RegisterServeFlags(cmd *cobra.Command, config *server.Config) {
 	cmd.Flags().StringVar(&config.TelemetryEndpoint, "telemetry-endpoint", telemetry.DefaultEndpoint, "endpoint to which telemetry is reported, empty string to disable")
 	cmd.Flags().StringVar(&config.TelemetryCAOverridePath, "telemetry-ca-override-path", "", "TODO")
 	cmd.Flags().DurationVar(&config.TelemetryInterval, "telemetry-interval", telemetry.DefaultInterval, "approximate period between telemetry reports, minimum 1 minute")
+
+	// Flag for lookupWatchApi enabler
+	cmd.Flags().BoolVar(&config.LookupWatchApiEnable, "lookup-watch-api-enable", false, "enable lookupWatchApi")
```

Comment:
This should be marked as an ALPHA feature

Reply:
Done I renamed the configuration variable to 
```
cmd.Flags().BoolVar(&config.LookupWatchApiEnable, "ALPHA.lookup-watch-api-enable", false, "enable lookupWatchApi")
```

Reply:
Change to `--enable-alpha-lookup-watch-api`

Reply:
Also indicate in the description that the API is in Alpha and subject to breaking

Reply:
done